hubblestack/daemon.py

```diff
@@ -253,7 +253,8 @@ def schedule():
             log.debug('Executing scheduled function {0}'.format(func))
             jobdata['last_run'] = time.time()
             ret = __salt__[func](*args, **kwargs)
-            log.debug('Job returned:\n{0}'.format(ret))
+            if __opts__['log_level'] == 'debug':
```

Comment:
If a lot of file events have accumulated over some time, 'ret' object becomes very large. In DME's case, fim runs after 200 seconds. On doing very high number of file operations in that time, the 'ret' object grows to a size of 12MB. Even though the object is not printed, hubble's memory increases continuously because of this function call. So putting it under if statement.